plugins/woocommerce/client/blocks/assets/js/blocks/add-to-cart-with-options/variation-selector/attribute-options/block.json

```diff
@@ -19,7 +19,29 @@
 	"$schema": "https://schemas.wp.org/trunk/block.json",
 	"supports": {
 		"inserter": false,
-		"interactivity": true
+		"interactivity": true,
+		"__experimentalStyle": {
+			"elements": {
+				"button": {
+					"spacing": {
+						"padding": {
+							"top": "0.25em",
+							"bottom": "0.25em",
+							"left": "0.75em",
+							"right": "0.75em"
+						}
+					},
+					"typography": {
+						"fontSize": "var(--wp--preset--font-size--small)"
+					},
+					"border": {
+						"width": "1px",
+						"style": "solid",
+						"radius": "2px"
```

Comment:
This will override the theme defaults, is that what we want?

Reply:
I copied the CSS and moved it here, but it sounds good to me if we don't add more styling than necessary